src/View/UIView.php

```diff
@@ -17,12 +17,14 @@ class UIView extends View
     /**
      * Initialization hook method.
      *
+     * @param array $options
      * @return void
      */
-    public function initialize()
+    public function initialize(array $options = [])
```

Comment:
`View::initialize()` does not take any params, so this proposed change will generate errors.